src/SwcCompiler.ts

```diff
@@ -178,9 +185,7 @@ export class SwcCompiler implements Compiler {
     // TODO: Use the config
     const { root, fileNames, swcConfig } = await this.getModule(filename);
 
-    await this.reportErrors(async () => {
-      await Promise.all(fileNames.map((filename) => this.buildFile(filename, root, swcConfig)));
-    });
+    await this.reportErrors(Promise.allSettled(fileNames.map((filename) => this.buildFile(filename, root, swcConfig))));
```

Comment:
I wasn't entirely sure if this was a bug or not. Files will stop compiling as soon as one file fails with `Promise.all`, this was fixed so that the benchmarks could be run